enos/modules/test_e2e/main.tf

```
@@ -98,6 +99,7 @@ locals {
 
 resource "enos_local_exec" "run_e2e_test" {
   environment = {
+    E2E_TESTS                     = true,
```

What does it mean to set this to a boolean value? Should this be
```suggestion
    E2E_TESTS                     = "true",
```